diff --git a/init.lua b/init.lua
index 2ce1cdd..643f873 100644
--- a/init.lua
+++ b/init.lua
@@ -17,3 +17,32 @@ end
 
 require "lazy_setup"
 require "polish"
+
+vim.opt.listchars = {
+  eol = '⤶',
+  space = '.',
+  trail = '#',
+}
+
+local cmp_nvim_lsp = require "cmp_nvim_lsp"
+
+require("lspconfig").clangd.setup {
+  on_attach = on_attach,
+  capabilities = cmp_nvim_lsp.default_capabilities(),
+  cmd = {
+    "clangd",
+    "--offset-encoding=utf-16",
+  },
+}
+
+require'lspconfig'.pylsp.setup{
+  settings = {
+    pylsp = {
+      plugins = {
+        pycodestyle = {
+          maxLineLength = 100
+        }
+      }
+    }
+  }
+}
diff --git a/lua/plugins/astrocore.lua b/lua/plugins/astrocore.lua
index c99b395..a3f0ccb 100644
--- a/lua/plugins/astrocore.lua
+++ b/lua/plugins/astrocore.lua
@@ -1,5 +1,3 @@
-if true then return {} end -- WARN: REMOVE THIS LINE TO ACTIVATE THIS FILE
-
 -- AstroCore provides a central place to modify mappings, vim options, autocommands, and more!
 -- Configuration documentation can be found with `:h astrocore`
 -- NOTE: We highly recommend setting up the Lua Language Server (`:LspInstall lua_ls`)
@@ -27,7 +25,7 @@ return {
     -- vim options can be configured here
     options = {
       opt = { -- vim.opt.<key>
-        relativenumber = true, -- sets vim.opt.relativenumber
+        relativenumber = false, -- sets vim.opt.relativenumber
         number = true, -- sets vim.opt.number
         spell = false, -- sets vim.opt.spell
         signcolumn = "auto", -- sets vim.opt.signcolumn to auto
@@ -62,6 +60,7 @@ return {
         -- tables with just a `desc` key will be registered with which-key if it's installed
         -- this is useful for naming menus
         ["<Leader>b"] = { desc = "Buffers" },
+        ["<Leader>p"] = { ":e#<cr>", desc = "Toggle" },
         -- quick save
         -- ["<C-s>"] = { ":w!<cr>", desc = "Save File" },  -- change description but the same command
       },
diff --git a/lua/plugins/astroui.lua b/lua/plugins/astroui.lua
index 737a2fb..1aa80e0 100644
--- a/lua/plugins/astroui.lua
+++ b/lua/plugins/astroui.lua
@@ -1,5 +1,3 @@
-if true then return {} end -- WARN: REMOVE THIS LINE TO ACTIVATE THIS FILE
-
 -- AstroUI provides the basis for configuring the AstroNvim User Interface
 -- Configuration documentation can be found with `:h astroui`
 -- NOTE: We highly recommend setting up the Lua Language Server (`:LspInstall lua_ls`)
@@ -11,7 +9,7 @@ return {
   ---@type AstroUIOpts
   opts = {
     -- change colorscheme
-    colorscheme = "astrodark",
+    colorscheme = "nightfly",
     -- AstroUI allows you to easily modify highlight groups easily for any and all colorschemes
     highlights = {
       init = { -- this table overrides highlights in all themes
diff --git a/lua/plugins/user.lua b/lua/plugins/user.lua
index 121089e..d46396f 100644
--- a/lua/plugins/user.lua
+++ b/lua/plugins/user.lua
@@ -1,5 +1,3 @@
-if true then return {} end -- WARN: REMOVE THIS LINE TO ACTIVATE THIS FILE
-
 -- You can also add or configure plugins by creating files in this `plugins/` folder
 -- Here are some examples:
 
@@ -82,4 +80,6 @@ return {
       )
     end,
   },
+  { "bluz71/vim-nightfly-colors" },
+  { "LhKipp/nvim-nu" }
 }
