diff --git a/init.lua b/init.lua
index 424e48d..e494d2c 100644
--- a/init.lua
+++ b/init.lua
@@ -25,3 +25,21 @@ end
 
 require "lazy_setup"
 require "polish"
+
+vim.opt.listchars = {
+  eol = '⤶',
+  space = '.',
+  trail = '#',
+}
+
+require'lspconfig'.pylsp.setup{
+  settings = {
+    pylsp = {
+      plugins = {
+        pycodestyle = {
+          maxLineLength = 100
+        }
+      }
+    }
+  }
+}
diff --git a/lua/lazy_setup.lua b/lua/lazy_setup.lua
index 4d9552f..34607cd 100644
--- a/lua/lazy_setup.lua
+++ b/lua/lazy_setup.lua
@@ -15,7 +15,7 @@ require("lazy").setup({
   { import = "plugins" },
 } --[[@as LazySpec]], {
   -- Configure any other `lazy.nvim` configuration options here
-  install = { colorscheme = { "astrotheme", "habamax" } },
+  install = { colorscheme = { "astrotheme", "habamax", "monokai" } },
   ui = { backdrop = 100 },
   performance = {
     rtp = {
diff --git a/lua/plugins/astrocore.lua b/lua/plugins/astrocore.lua
index 03d7441..a029977 100644
--- a/lua/plugins/astrocore.lua
+++ b/lua/plugins/astrocore.lua
@@ -1,5 +1,3 @@
-if true then return {} end -- WARN: REMOVE THIS LINE TO ACTIVATE THIS FILE
-
 -- AstroCore provides a central place to modify mappings, vim options, autocommands, and more!
 -- Configuration documentation can be found with `:h astrocore`
 -- NOTE: We highly recommend setting up the Lua Language Server (`:LspInstall lua_ls`)
@@ -40,7 +38,7 @@ return {
     -- vim options can be configured here
     options = {
       opt = { -- vim.opt.<key>
-        relativenumber = true, -- sets vim.opt.relativenumber
+        relativenumber = false, -- sets vim.opt.relativenumber
         number = true, -- sets vim.opt.number
         spell = false, -- sets vim.opt.spell
         signcolumn = "yes", -- sets vim.opt.signcolumn to yes
diff --git a/lua/plugins/astroui.lua b/lua/plugins/astroui.lua
index 737a2fb..ce0aee6 100644
--- a/lua/plugins/astroui.lua
+++ b/lua/plugins/astroui.lua
@@ -1,5 +1,3 @@
-if true then return {} end -- WARN: REMOVE THIS LINE TO ACTIVATE THIS FILE
-
 -- AstroUI provides the basis for configuring the AstroNvim User Interface
 -- Configuration documentation can be found with `:h astroui`
 -- NOTE: We highly recommend setting up the Lua Language Server (`:LspInstall lua_ls`)
@@ -11,16 +9,16 @@ return {
   ---@type AstroUIOpts
   opts = {
     -- change colorscheme
-    colorscheme = "astrodark",
+    colorscheme = "monokai",
     -- AstroUI allows you to easily modify highlight groups easily for any and all colorschemes
-    highlights = {
-      init = { -- this table overrides highlights in all themes
-        -- Normal = { bg = "#000000" },
-      },
-      astrodark = { -- a table of overrides/changes when applying the astrotheme theme
-        -- Normal = { bg = "#000000" },
-      },
-    },
+    -- highlights = {
+    --   init = { -- this table overrides highlights in all themes
+    --     -- Normal = { bg = "#000000" },
+    --   },
+    --   monokai = { -- a table of overrides/changes when applying the astrotheme theme
+    --     -- Normal = { bg = "#000000" },
+    --   },
+    -- },
     -- Icons can be configured throughout the interface
     icons = {
       -- configure the loading of the lsp in the status line
